dotnet/samples/Concepts/ChatCompletion/OpenAI_FunctionCalling.cs

```diff
@@ -67,6 +68,25 @@ public async Task AutoInvokeKernelFunctionsWithComplexParameterAsync()
         Console.WriteLine(chatPromptResult);
     }
 
+    [Fact]
+    public async Task AutoInvokeLightPluginAsync()
+    {
+        // Create a kernel with MistralAI chat completion and WeatherPlugin
+        Kernel kernel = CreateKernelWithPlugin<LightPlugin>();
+        kernel.FunctionInvocationFilters.Add(new FunctionFilterExample(this.Output));
+
+        // Invoke chat prompt with auto invocation of functions enabled
+        const string ChatPrompt = """
+            <message role="user">Turn on the light?</message>
+        """;
+        var executionSettings = new OpenAIPromptExecutionSettings { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions };
+        var chatSemanticFunction = kernel.CreateFunctionFromPrompt(
+            ChatPrompt, executionSettings);
```

Comment:
Since we are not using multiple user messages in the example, just passing the text might be enough.

```suggestion
        var executionSettings = new OpenAIPromptExecutionSettings { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions };
        var chatSemanticFunction = kernel.CreateFunctionFromPrompt("Turn on the light?", executionSettings);
```

Reply:
I'm following the same pattern as all of the other samples in this file so I will leave it as is